packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1218,6 +1218,12 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
             nextRenderExpirationTime = NoWork;
           }
           requestWork(root, expirationTime);
+          if (
+            !isWorking &&
+            root === nextRoot &&
+            expirationTime < nextRenderExpirationTime
+          ) {
```

Comment:
Nothing here?

Reply:
No idea what my brain was thinking then. Sorry, pushed the code.